### DISCUSSION TITLE
Merging Feature Branches

### DISCUSSION THREAD

**Discussion by JN889** [namita2024/Tafe24S107#6] [General]
I am in process of merging the feature_MortgageCalculator and Development_Branch. Checkout the changes made and then the feature_CurrencyCalculator can be merged once updated with fixes.

**Comment by namita2024** [comment]
Thanks for letting me know, i am about to merge my feature branch now, once i finish , i will let you know

**Comment by namita2024** [comment]
merge been completed , you can start with release branch, thanks
